src/pymatgen/symmetry/groups.py

```diff
@@ -18,7 +18,6 @@
 from monty.design_patterns import cached_class
 from monty.serialization import loadfn
 
-from pymatgen.core.operations import SymmOp
```

Comment:
looks like `pre-commit` hasn't been updated to recognize the `TC004` rule and would reposition this import

Reply:
bump ruff hook in pre-commit seems to resolve this 7825975594f95c9d9c73efa9a5c4a58e0e43419e